### ISSUE TITLE
Warning in console mode (ddg.annoate.inside)

### ISSUE THREAD

**Issue by tfjmp** [End-to-end-provenance/RDataTracker#243]
Warning message:
In .ddg.insert.error.message(error.msg) :
  No binding for ddg.annotate.inside . DDG may be incorrect!

ddg.init()
x <- 1
y <- 2
z <- x + y
ddg.save()
<--- warning is here.

**Comment by blernermhc** [comment]
I do not see that error on my Mac.  What version of RDataTracker are you using?

```
> library(RDataTracker)
> ddg.init()
> x <- 1
> y <- 2
> z <- x + y
> ddg.save()
[1] "Saving ddg.txt in /Users/blerner/Documents/Process/DataProvenance/workspace/eep-usecases/eep-usecases/src/ddg"
[1] "Saving ddg.json in /Users/blerner/Documents/Process/DataProvenance/workspace/eep-usecases/eep-usecases/src/ddg"
```

**Comment by blernermhc** [comment]
Fixed in 54cef82f649304633f31487cc1f2aae1c8cd751c
